compiler/src/test/java/com/squareup/anvil/compiler/dagger/MembersInjectorGeneratorTest.kt

```diff
@@ -122,11 +134,13 @@ public final class InjectClass_MembersInjector implements MembersInjector<Inject
       package com.squareup.test
       
       import javax.inject.Inject
+      import javax.inject.Named
       
       typealias StringList = List<String>
       
       class InjectClass {
         @Inject lateinit var string: String
+        @Named("qualified") @Inject lateinit var qualifiedString: String
```

Comment:
The updated test passed even without your changes to fix the issue. I've added this check:
```kotlin
val namedAnnotation = membersInjector.staticInjectMethod("qualifiedString").annotations
  .single { it.annotationClass == Named::class }
assertThat(namedAnnotation.getValue<String>()).isEqualTo("qualified")
```